### ISSUE TITLE
How to get multiple values from external value ?

### ISSUE THREAD

**Issue by tinkerbotfoo** [spurin/xmldataset#9]
consider the following profile ., i would need to get both the probe name & probe id from the ancestor level into the data. However only one value (probe id) in this case gets propagated, probe name is ignored. Is it a know limitation or i am not getting the profile syntax correct ?

```
profile= """
prtg
    sensortree
        nodes
            group
                probenode
                    name = external_dataset:probenode,name:probenode_name
                    id = external_dataset:probenode,name:probenode_id
                    device    
                        name    = dataset:nodes
                        tags    = dataset:nodes
                        host    = dataset:nodes
                        active  = dataset:nodes,name:active_status
                        __EXTERNAL_VALUE__ = probenode:probenode_name:nodes
                        __EXTERNAL_VALUE__ = probenode:probenode_id:nodes
"""    
```

**Comment by tinkerbotfoo** [comment]
Thanks to your well written test ( def test_external_data_custom_name(self):.
 I was able to figure out how to do it ...

once i put the probe name & probe id together in the same external value ., it works

```
profile= """
prtg
    sensortree
        nodes
            group
                probenode
                    name = external_dataset:probenode,name:probenode_name
                    id = external_dataset:probenode,name:probenode_id
                    device    
                        name    = dataset:nodes
                        tags    = dataset:nodes
                        host    = dataset:nodes
                        active  = dataset:nodes,name:active_status
                        __EXTERNAL_VALUE__ = probenode:probenode_name:nodes probenode:probenode_id:nodes                        
"""   
```

**Comment by spurin** [comment]
Awesome, given that you had an issue with this it's probably worth the documentation being updated to clarify this functionality, not everyone will look into the test cases like you have.  I'll leave this open for pull requests or change myself in the future.  Thanks for highlighting.

**Comment by tinkerbotfoo** [comment]
i created the pull request #10 for this ... Thanks again for writing an awesome library and tests.

**Comment by spurin** [comment]
Merged into 1.0.2, also credited you on docs front page.  Thanks @tinkerbotfoo
